junit-platform-console/src/main/java/org/junit/platform/console/options/AvailableOptions.java

```diff
@@ -70,8 +73,18 @@
 		disableAnsiColors = parser.accepts("disable-ansi-colors",
 			"Disable ANSI colors in output (not supported by all terminals).");
 
-		hideDetails = parser.accepts("hide-details",
-			"Hide details while tests are being executed. Only show the summary and test failures.");
+		// TODO Remove "hide-details" before final release. Only here to make "legacy" Gradle plugin happy.
+		// If removed, an old Gradle:junitPlatformTest will report:
+		//   Exception in thread "main" joptsimple.UnrecognizedOptionException: hide-details is not a recognized option
+		parser.accepts("hide-details", "@Deprecated. Use '--details none' instead.");
+
+		List<String> names = stream(Details.values()).map(d -> d.name().toLowerCase()).collect(toList());
```

Comment:
I took another road -- and am happy with now. See https://github.com/junit-team/junit5/pull/524/commits/55f7fcbbf864cf4f40d908be05df0adc22c0b2b2